### PR TITLE
[MIN-1597] Minswap syncer for query price for v1, v2 and stableswap pool.

### PR DESCRIPTION
**Pull request by hieupnh**

### CODE REVIEW

File: src/types/tx.internal.ts [hunk 1]
```diff
@@ -14,4 +16,26 @@ export type TxHistory = {
   txIndex: number;
   blockHeight: number;
   time: Date;
+}
+
+export namespace Value {
+  export function fromOgmiosValue(ogmiosValue: OgmiosSchema.Value): Value {
+    const value: Value = [];
+    for (const [cyrencySymbol, tokenNameMap] of Object.entries(ogmiosValue)) {
```
Comment (h2physics) [inline src/types/tx.internal.ts:24]:
```suggestion
    for (const [currencySymbol, tokenNameMap] of Object.entries(ogmiosValue)) {
```

File: package.json [hunk 2]
```diff
@@ -42,20 +49,27 @@
   "homepage": "https://github.com/minswap/sdk#readme",
   "dependencies": {
     "@blockfrost/blockfrost-js": "^5.5.0",
+    "@cardano-ogmios/client": "^6.5.0",
+    "@cardano-ogmios/schema": "^6.7.0",
```
Comment (longngn) [inline package.json:53]: need to be the same and similar to Ogmios in Docker files

File: package.json [hunk 3]
```diff
@@ -42,20 +49,27 @@
...
   "devDependencies": {
     "@eslint/compat": "^1.1.1",
     "@eslint/js": "^9.9.1",
+    "@jest/globals": "^29.7.0",
+    "@prisma/client": "^5.19.1",
     "@types/big.js": "^6.2.2",
     "@types/eslint__js": "^8.42.3",
     "@types/jest": "^29.5.12",
     "@types/json-bigint": "^1.0.4",
+    "@types/node": "^22.5.5",
```
Comment (longngn) [inline package.json:72]: should be 20.x.x (because we use node 20)

File: src/syncer/repository/postgres-reposiotry.ts [hunk 4]
```diff
@@ -0,0 +1,342 @@
+import * as OgmiosSchema from "@cardano-ogmios/schema";
```
Comment (longngn) [inline src/syncer/repository/postgres-reposiotry.ts:1]: file name typo
reposiotry => repository

**Review by longngn — CHANGES_REQUESTED** [review]

**Comment by longngn** [comment]
Test import lib like this:

1. Create new node project
2. Import by local dir like this: `"@minswap/sdk": "local/path/to/sdk/build/dir"`
3. Test if we can import and use `@minswap/sdk` as a lib

**Comment by lednhatkhanh** [comment]
@longngn @hieupnh I'm fixing this

**Review by h2physics — APPROVED** [review]

**Pull request merged**